src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -159,6 +166,7 @@ export class ConnectionWidget extends lifecycle.Disposable {
 	public createConnectionWidget(container: HTMLElement, authTypeChanged: boolean = false): void {
 		this._serverGroupOptions = [this.DefaultServerGroup];
 		this._serverGroupSelectBox = new SelectBox(this._serverGroupOptions.map(g => g.name), this.DefaultServerGroup.name, this._contextViewService, undefined, { ariaLabel: this._serverGroupDisplayString });
+		this._register(this._serverGroupSelectBox);
```

Comment:
You can shorten these by just adding the _register call where you create it. Not a huge deal if you think it's cleaner like this though

Reply:
I'd prefer to keep it this way, easier to read.

Reply:
Maybe, but keep in mind that nearly all of the code does it inline and so it's easy to miss seeing that when skimming the code. I almost left a couple of comments asking why you hadn't added them before I noticed it was the line below. (and given how verbose these lines are anyways I'm not sure it's really that much harder to read)